OptionD-ExpandedSimple/.funcignore

```diff
@@ -0,0 +1,7 @@
+*.js.map
```

Comment:
> Uses an app root object for all function registrations

This sort of feedback is very helpful. It was mentioned here as well so I might just update all options to do that: https://github.com/ejizba/func-nodejs-prototype/issues/1